manim/mobject/svg/text_mobject.py

```diff
@@ -56,6 +56,7 @@ def construct(self):
 
 import manimpango
 import numpy as np
+from colour import Color
```

Comment:
NIT: you have imported this just for adding typings so this should be under the check for https://docs.python.org/3/library/typing.html#typing.TYPE_CHECKING